examples/quality_classifier_example.py

```diff
@@ -60,52 +60,16 @@ def attach_args(
         formatter_class=argparse.ArgumentDefaultsHelpFormatter
     ),
 ):
-    parser.add_argument(
-        "--scheduler-address",
-        type=str,
-        default=None,
-        help="Address to the scheduler of a created dask cluster. If not provided"
-        "a single node LocalCUDACluster will be started.",
-    )
-    parser.add_argument(
-        "--scheduler-file",
-        type=str,
-        default=None,
-        help="Path to the scheduler file of a created dask cluster. If not provided"
-        " a single node LocalCUDACluster will be started.",
-    )
-    parser.add_argument(
-        "--nvlink-only",
-        action="store_true",
-        help="Start a local cluster with only NVLink enabled."
-        "Only applicable when protocol=ucx and no scheduler file/address is specified",
-    )
-    parser.add_argument(
-        "--protocol",
-        type=str,
-        default="ucx",
-        help="Protcol to use for dask cluster"
-        "Note: This only applies to the localCUDACluster. If providing an user created "
-        "cluster refer to"
-        "https://docs.rapids.ai/api/dask-cuda/stable/api.html#cmdoption-dask-cuda-protocol",  # noqa: E501
-    )
-    parser.add_argument(
-        "--rmm-pool-size",
-        type=str,
-        default="14GB",
-        help="Initial pool size to use for the RMM Pool Memory allocator"
-        "Note: This only applies to the localCUDACluster. If providing an user created "
-        "cluster refer to"
-        "https://docs.rapids.ai/api/dask-cuda/stable/api.html#cmdoption-dask-cuda-rmm-pool-size",  # noqa: E501
-    )
-    parser.add_argument("--enable-spilling", action="store_true")
-    parser.add_argument("--set-torch-to-use-rmm", action="store_true")
-    parser.add_argument(
-        "--device",
-        type=str,
-        default="gpu",
-        help="Device to run the script on. Either 'cpu' or 'gpu'.",
-    )
+    argumentHelper = ArgumentHelper(parser)
+
+    argumentHelper.add_device()
+    argumentHelper.add_enable_spilling()
+    argumentHelper.add_nvlink_only()
+    argumentHelper.add_protocol()
+    argumentHelper.add_rmm_pool_size()
+    argumentHelper.add_scheduler_address()
+    argumentHelper.add_scheduler_file()
+    argumentHelper.add_set_torch_to_use_rmm()
```

Comment:
I suspect this has a high overlap with add_distributed_args

Reply:
Hi @ayushdg , you are right. There is a high overlap with add_distributed_args in this file, `quality_classifier_example`, and also in `domain_classifier_example.py`.

If you do not mind, because the overlap is not 100% and it requires a deeper analysis if I can replace those calls with `add_distributed_args`, I would appreciate if we could pospone this change to a near future pull request. 

I hope you are happy with the above. Thanks!